olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.h

```diff
@@ -60,6 +61,12 @@ class PartitionsRepository {
   static model::Partition PartitionFromSubQuad(
       const model::SubQuad& sub_quad, const std::string& partition);
 
+  static PartitionsResponse GetAggregatedTile(
```

Comment:
Why do we need a vector as output? One partition is not enough?

Reply:
I have chosen PartitionsResponse because it can contain desired partition and can provide error things.
But yes, vector is overkill here.

Maybe, it makes sense to return QuadTreeIndex here? So in VersionedLayerClient we can at least differ online/cache errors from tile not found in quad tree case.

Reply:
Creating a PartitionResponse, means `Response<model::Partition>`, is probably the least problematic part.
If this is the only issue then please add an alias and lets only return one single model::Partition.
Returning QuadTreeIndex means that all the methods using this GetAggregatedTile() actually need to implement the logic themselves. Then there is no point anymore to call it GetAggregatedTile() but rather GetQuadTreeIndex().